Monika After Story/game/script-stories.rpy

```diff
@@ -447,6 +447,44 @@ label mas_story_immortal_love:
     m 1hua "Thanks for listening~"
     return
 
+init 5 python:
+    addEvent(Event(persistent._mas_story_database,eventlabel="mas_story_self_hate",
+        prompt="Self-hate",unlocked=False),eventdb=store.mas_stories.story_database)
+
+label mas_story_self_hate:
+
+    m 1esa "Once, there were two lovers who lived together."

```

Comment:
Can combine this line and the next into "Once, there were two lovers who had been living together for a very long time."